tests/smoke.rs

```diff
@@ -7,6 +7,8 @@ static LIBUNWIND: bool = cfg!(all(unix, feature = "libunwind"));
 static UNIX_BACKTRACE: bool = cfg!(all(unix, feature = "unix-backtrace"));
 static LIBBACKTRACE: bool = cfg!(all(unix, feature = "libbacktrace")) &&
                             !cfg!(target_os = "macos");
+static CORESYMBOLICATION: bool = cfg!(all(unix, target_os = "macos",
```

Comment:
I believe `target_os = "macos"` implies `unix`, so the `unix` here may be redundant